docs/user_guides/tls_proxy.md

```diff
@@ -429,6 +429,7 @@ They will creat 9 files about ca.key, ca.pem, ca.srl, server.key, server.pem, se
 ## Modify config
 Modify tlsEnabled to true and the file path in config/milvus.yaml.  
 Server need server.pem, server.key and ca.pem. Client need client.pem, client.key, ca.pem.
```

Comment:
We'd better add two parts to describe how to config:
- server-side
- client-side